azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestBase.java

```diff
@@ -170,4 +171,8 @@ protected void succeedJobs(final String... jobs) {
     }
   }
 
+  protected void assertEvents(final Type... types) {
```

Comment:
"I guess my thinking there is that you already know for example that assertEvents will check against the "current eventCollector" or that assertStatus will check against the "current flow" withouth having to every time echo the reference to event collector or flow runner."
I see.
But the contract wasn't clear to me when I read it and I think I will have the same reaction the next time I read this code.
My personal experience is that my mental overhead goes up when I see code that relies on inheritance typically.